nucliadb/nucliadb/reader/api/v1/learning_config.py

```diff
@@ -32,7 +34,7 @@
     name="Download the Knowledege Box model",
     description="Download the trained model or any other generated file as a result of a training task on a Knowledge Box.",  # noqa
     response_model=None,
-    tags=["Training"],
+    tags=["Models"],
```

Comment:
NIT: Can we use `LLM models` or something else to not get mixed up with pydantic/API models?